evap/rewards/tools.py

```diff
@@ -25,11 +26,20 @@
 
 @login_required
 @transaction.atomic
-def save_redemptions(request, redemptions: Dict[int, int]):
+def save_redemptions(request, redemptions: Dict[int, int], previous_reward_points):
+    """Saves the redemptions in the given dict. Returns True if the redemptions were saved successfully, False otherwise."""
```

Comment:
You could add a type annotation for `previous_reward_points` (for `request`, it is more complicated, so it can stay as is)

Secondly, this always returns `True`, or raises

Reply:
Jup. Simply forgot to remove it